### PR TITLE
Update ValueError text, example files

### PR DESCRIPTION
**Pull request by tekktrik**
Addresses Issue #29 by changing the ValueError raised to be clearer, and to update the example files to get and set multiple memory addresses with slices correctly.

### CODE REVIEW

File: examples/fram_i2c_simpletest.py [hunk 1]
```diff
@@ -36,6 +36,6 @@
 ##       a buffer the size of slice, which may cause
 ##       problems on memory-constrained platforms.
 
-# values = list(range(100)) # or bytearray or tuple
-# fram[0] = values
-# print(fram[0:99])
+values = list(range(100))  # or bytearray or tuple
+fram[0:100] = values
```
Comment (FoamyGuy) [inline examples/fram_i2c_simpletest.py]: Lets keep the writes commented out in the example that way it's harder for someone to unknowingly cause wear on their device by running the example unmodified.

File: examples/fram_spi_simpletest.py [hunk 2]
```diff
@@ -27,6 +27,6 @@
 ##       a buffer the size of 'length', which may cause
 ##       problems on memory-constrained platforms.
 
-# values = list(range(100)) # or bytearray or tuple
-# fram[0] = values
-# print(fram[0:99])
+values = list(range(100))  # or bytearray or tuple
+fram[0:100] = values
```
Comment (FoamyGuy) [inline examples/fram_spi_simpletest.py]: Same with these ones lets keep them commented out.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
Thanks @tekktrik The error text looks good to me now.

have one other small request.

**Comment by tekktrik** [comment]
Thanks for the feedback!  Resubmitted changes for review.

**Review by FoamyGuy — APPROVED** [review]
Looks good to me

**Pull request merged**